### PR TITLE
Polish in CONTRIBUTING.md

### PR DESCRIPTION
**Pull request by skytin1004**
I've cleaned up the text to make it more readable in CONTRIBUTING.md.

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -67,9 +67,9 @@ docker run apache/kibble-dev pytest
 ## Testing
 
 Apache Kibble project uses [pytest](https://docs.pytest.org/en/stable/) for running testing. Writing
-good test helps us avoid regression and unexpected issues.
+the good test helps us avoid regression and unexpected issues.
```
Comment (sharanf) [inline CONTRIBUTING.md]: Should be 'Writing a good test.. or *Writing good tests helps ....'

**Review by sharanf — CHANGES_REQUESTED** [review]

**Comment by sharanf** [comment]
> I've cleaned up the text to make it more readable in CONTRIBUTING.md.

Hi @skytin1004 thanks very much for the contribution and my apologies for not reviewing this earlier. I've highlighted a minor grammar change and once fixed will be good to merge.

File: CONTRIBUTING.md (hunk 1 above)
Comment (skytin1004) [reply CONTRIBUTING.md]: Thank you for the review @sharanf , I modified based on your review.

"Writing good tests help us avoid regression and unexpected issues."

**Review by sharanf — APPROVED** [review]
Thanks for doing the updates.

**Pull request merged**